subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/context/DialogInteractionContext.kt

```diff
@@ -0,0 +1,25 @@
+package com.avito.android.test.context
+
+import android.view.View
+import androidx.test.espresso.matcher.RootMatchers
+import androidx.test.espresso.matcher.ViewMatchers
+import com.avito.android.test.InteractionContext
+import com.avito.android.test.SimpleInteractionContext
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers
+
+class DialogInteractionContext(
+    private val matcher: Matcher<View>,
+) : SimpleInteractionContext(
+    matcher = matcher,
+    rootMatcher = RootMatchers.isDialog()
+) {
+
+    override fun provideChildContext(matcher: Matcher<View>): InteractionContext =
+        DialogInteractionContext(
```

Comment:
As I understand the Dialog should create the SimpleInteractionContext for a child

Reply:
Fixed